### ISSUE TITLE
Issue with multiline template strings

### ISSUE THREAD

**Issue by overengineered** [sindresorhus/execa#901]
I found this example in [documentation](https://github.com/sindresorhus/execa/blob/main/docs/scripts.md#multiline-commands):

```
await $`npm run build
	--example-flag-one
	--example-flag-two`
```

I tried running:

```
await $({ verbose: true })`mkdir -p 
${"dir1"}`.pipeStdout?.(process.stdout);
```

Note the space after the `-p`. Execa verbose showed this as command:

```
[22:20:12.537] mkdir -p "
dir1"
```

On MacOS this actually created a folder with newline as the first name character. I tried

```
await $({ verbose: true })`mkdir -p \
${"dir1"}`.pipeStdout?.(process.stdout);
```

This created the expected folder simply named "dir1". I'm using version 8.0.1

**Comment by ehmicky** [comment]
Hi @overengineered,

This feature has been implemented but has not been released yet. We're working on it, but it might take a few more weeks. Thanks for your interest in the feature!

**Comment by ehmicky** [comment]
This feature has been just released in Execa 9.0.0. Please see (and share!) the [release post](https://medium.com/@ehmicky/execa-9-release-d0d5daaa097f) and the [changelog](https://github.com/sindresorhus/execa/releases/tag/v9.0.0).
